src/data/taskList.ts

```diff
@@ -67,7 +68,6 @@ export const finalBusinessCaseTag = (intake: SystemIntakeForm) => {
     case 'BIZ_CASE_CHANGES_NEEDED':
       return 'CANNOT_START';
     case 'BIZ_CASE_FINAL_SUBMITTED':
-    case 'READY_FOR_GRT':
```

Comment:
Step 4 doesn't need the `Completed` tag when the intake is `READY_FOR_GRT`?

Reply:
the wireframes don't show that as the case, when the Ready for GRT option is selected buy the reviewer it shouldn't even mark step 3 as complete so it felt wrong to have it mark step 4 as complete

Reply:
This is what it looks like with that line added back in.

![image](https://user-images.githubusercontent.com/68014929/99537964-4e386600-297a-11eb-8732-04991f242d90.png)

Reply:
I see. I must be crossing wires with another status